app/globals.css

```
@@ -257,7 +257,10 @@
   }
 
   .dashboard-recent-files {
-    @apply h-full flex flex-col rounded-[20px] xl:h-163.5 overflow-auto bg-white p-5 xl:p-7;
+    @apply flex flex-col rounded-[20px] bg-white p-5 xl:p-7 transition-all duration-300;
+    height: fit-content;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)